### PR TITLE
Metabolomics - Update of "LCMS preprocessing with XCMS"

### PR DESCRIPTION
**Pull request by melpetera**
Hi there,

Here are a few updates regarding the "LCMS preprocessing with XCMS" training material.
It is not much, mainly it is minor changes suggested by users. Thanks a lot to all of them for there feedbacks!

Thank you in advance for the review and potential additional suggestions one may have. 

Best regards,
Mélanie

### CODE REVIEW

**Review by hexylena — APPROVED** [review]

**Comment by melpetera** [comment]
Thank you @hexylena for the review!
I am not sure what should be done next? I see that there is an error to access one picture but I did not change anything about this file, and the line seems correct to me (but I am no expert here so I don't know).

File: topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md [hunk 1]
```diff
@@ -50,23 +50,23 @@ the results obtained may not be reliable enough for biological interpretation du
...
 
 We chose a subset of 12 samples, composed of 6 biological samples, 3 quality-control pooled samples ('QC pools' - mix of all
-biological samples) and 3 blank samples ('blanks' - injection solvent).
+biological samples) and 3 blank samples ('blanks' - solvent injection).
 
 To analyze these data, we will then follow a Galaxy workflow
 developed by the [Wokflow4metabolomics group](http://workflow4metabolomics.org/) ({% cite Giacomoni2014 %}, {% cite Guitton2017 %}).
 
-[![The full tutorial workflow](../../images/lcmspreproc_wf.png)](../../images/lcmspreproc_wf.png)
+[![The full tutorial workflow represented as boxes linked with arrows, each box being labeled with the corresponding galaxy tool name](../../images/lcmspreproc_wf.png)](../../images/lcmspreproc_wf.png "The tutorial workflow")
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md:58]: I guess in a future version we can make this more automatic somehow, so you don't have to do it manually.

[the remark this answers is not in the document]
Comment (hexylena) [reply topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md:58]: @melpetera if you apply this change, then we can merge this PR.

[the remark this answers is not in the document]
Comment (melpetera) [reply topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md]: thank you again for the review and suggestion!
I am not too familiar with GitHub reviewing actions. I hope I made the right move? It seems I validated the suggestion (or at least that was what I intended to do).

**Review by hexylena — APPROVED** [review]

**Comment by melpetera** [comment]
Thank you very much @hexylena for your time and review!

**Pull request merged**